core/network/src/protocol.rs

```diff
@@ -433,7 +433,7 @@ impl<B: BlockT, S: NetworkSpecialization<B>, H: ExHashT> Protocol<B, S, H> {
 		}
 
 		debug!(target: "sync", "{} clogging messages:", clogging_messages.len());
-		for msg_bytes in clogging_messages {
+		for msg_bytes in clogging_messages.take(5) {
```

Comment:
https://www.youtube.com/watch?v=vmDDOFXSgAs

why five?

Reply:
Not to spam logs too much. 5 should be enough to identify what types of messages are clogging the queue.